CODEOWNERS

```diff
@@ -8,4 +8,4 @@
 # the repo. Unless a later match takes precedence,
 # @user will be requested for review when someone opens
 # a pull request.
-*       @sonic-kernel
+*       @sonic-net/sonic-linux-kernel-maintainer
```

Comment:
This could go into a separate merge/pull request and could be submitted shortly, as it’s not WIP.

Reply:
Agreed, opened #319 for this and other small changes.